ros2nodl/setup.py

```diff
@@ -0,0 +1,34 @@
+from setuptools import find_packages, setup
+
+package_name = 'ros2nodl'
+
+setup(
+    name=package_name,
+    version='0.0.1',
+    packages=find_packages(exclude=['test']),
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    author='Ted Kern',
+    author_email='ted.kern@canonical.com',
+    maintainer='Ubuntu Robotics',
+    maintainer_email='ubuntu-robotics@lists.launchpad.net',
```

Comment:
Let's just have the author instead of both author and maintainer, and leave it as Ubuntu Robotics, please. I think nodl is that way right now as well-- can you update it too, please?

Reply:
Fixed in b8c715a